modules/hello-world/pages/overview.adoc

```diff
@@ -98,9 +89,9 @@ https://docs.couchbase.com/sdk-api/couchbase-python-client/[API Docs^]
 .What's Hot?
 
 [.content]
-The Couchbase Python SDK 3.0 is a complete rewrite of the API, reducing the number of overloads to present a simplified surface area, and adding support for future Couchbase Server features like Collections and Scopes (available in Couchbase Server 6.5 & 6.6 as a xref:concept-docs:collections.adoc[developer preview]).
+The Couchbase Python SDK 3.x is a complete rewrite of the API, reducing the number of overloads to present a simplified surface area, and adding support for future Couchbase Server features like Collections and Scopes (available in Couchbase Server 6.5 & 6.6 as a xref:concept-docs:collections.adoc[developer preview]).
```

Comment:
Q (possibly for @RichardSmedley) is "3.x" a convention we use in docs, or something that we'd need to put in a ticket to update before release?